packages/components/src/components/dropdown/dropdown.scss

```diff
@@ -300,4 +313,27 @@
             right: 16px;
         }
     }
+    &--filter {
+        margin-left: 1rem;
+        margin-top: 0;
+        max-width: fit-content;
+        @include mobile {
+            height: 4rem;
+        }
+    }
+}
+
+.icon_filter {
+    position: absolute;
+    left: 10px;
+    @include mobile {
+        left: 12px;
+    }
+}
+
+.dc-list-filter {
+    width: 17.6rem !important;
+    @include mobile {
```

Comment:
Dropdown shouldn't be used in mobile so we can omit `@include mobile { ... }`

Reply:
this one is a list and can be removed